src/app/zap-templates/zcl/data-model/chip/joint-fabric-datastore-cluster.xml

```diff
@@ -0,0 +1,370 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024-2025 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="DatastoreStateEnum" type="enum8">
+    <cluster code="0x0752"/>
+    <item name="Pending" value="0x00"/>
+    <item name="Committed" value="0x01"/>
+    <item name="DeletePending" value="0x02"/>
+    <item name="CommitFailed" value="0x03"/>
+  </enum>
+
+  <struct name="DatastoreStatusEntryStruct">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="State" type="DatastoreStateEnum" default="0x00" min="0x00" max="0x03"/>
+    <item fieldId="1" name="UpdateTimestamp" type="epoch_s"/>
+    <item fieldId="2" name="FailureCode" type="status" default="0"/>
+  </struct>
+
+  <struct name="DatastoreNodeKeySetEntry" apiMaturity="provisional" isFabricScoped="true">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="NodeID" type="node_id"/>
+    <item fieldId="1" name="GroupKeySetID" type="int16u"/>
+    <item fieldId="2" name="StatusEntry" type="DatastoreStatusEntryStruct"/>
+  </struct>
+
+  <enum name="DatastoreAccessControlEntryPrivilegeEnum" type="enum8">
+    <cluster code="0x0752"/>
+    <item name="View" value="0x01"/>
+    <item name="ProxyView" value="0x02"/>
```

Comment:
Why is ProxyView here?  It shouldn't be here (or in the spec).

Reply:
DatastoreAccessControlEntryPrivilegeEnum was mainly inspired by the existing AccessControlEntryPrivilegeEnum which has the "ProxyView". How should we proceed here? @chrisdecenzo

Reply:
Tracked by https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11620